### PR TITLE
E2e adding ConfigMaps tests

### LINKED ISSUE #110: Add e2e tests for ConfigMap default and constants
Add automated e2e tests for ConfigMap default and constants.

### PR DESCRIPTION
**Pull request by JusteenR**
**What this PR does / why we need it**:

- Tests that the configmaps defaults and constants are applied correctly
- Checks that the default values are applied and constants are not changed

**Which issue(s) this PR fixes**:

Fixes #110

### CODE REVIEW

**Comment by JusteenR** [comment]
Fixes #110

File: test/e2e/appsody_configmapsconst.go [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+	}
+
+	configMap.Data = updateData
+
+	err = f.Client.Update(goctx.TODO(), configMap)
+	if err != nil {
+		t.Log(err)
+	}
+
+	err = f.Client.Get(goctx.TODO(), types.NamespacedName{Name: "appsody-operator-constants", Namespace: namespace}, configMap)
```
[the remark this answers is not in the document]
Comment (navidsh) [reply test/e2e/appsody_configmapsconst.go]: I noticed a couple of instances of this in this file and the other test.

File: test/e2e/appsody_configmapsconst.go [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+
+	f := framework.Global
+
+	// Values to be loaded in the constants configmap
+	updateData := map[string]string{"jstack": `{"version": 1.0.0,"expose":true, "service":{"port": 3000,"type": NodePort}, "livenessProbe":{"failureThreshold": 8, "httpGet":{"path": /live, "port": 3000}, "initialDelaySeconds": 8, "periodSeconds": 2}, "readinessProbe":{"failureThreshold": 12, "httpGet":{"path": /ready, "port": 3000}, "initialDelaySeconds": 5, "periodSeconds": 2, "timeoutSeconds": 1}}`}
+	configMap := &corev1.ConfigMap{}
+
+	err = f.Client.Get(goctx.TODO(), types.NamespacedName{Name: "appsody-operator-constants", Namespace: namespace}, configMap)
+	if err != nil {
+		t.Log(err)
```
[the remark this answers is not in the document]
Comment (JusteenR) [reply test/e2e/appsody_configmapsconst.go]: Good point, if there is an error I'll change it to `t.Fatal()`.

File: test/e2e/appsody_configmapsconst.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (JusteenR) [reply test/e2e/appsody_configmapsconst.go]: If we do not get the ConfigMap again the test will still pass, but the logger throws error messages that the object has been modified.

[the remark this answers is not in the document]
Comment (JusteenR) [reply test/e2e/appsody_configmapsconst.go]: You're right there is no need to to get the the ConfigMap again, I'll remove the extra calls to that function.

**Pull request merged**